narwhal/consensus/src/metrics.rs

```diff
@@ -36,6 +40,9 @@ pub struct ConsensusMetrics {
     /// * not_enough_support: when the leader certificate has been found but there was not enough support
     /// * elected: when the leader certificate has been found and had enough support
     pub leader_election: IntCounterVec,
+    /// The time it takes for a certificate from the moment it gets created
+    /// up to the moment it gets committed.
+    pub certificate_commit_latency: Histogram,
```

Comment:
Thanks for the feedback @asonnino !

> it would also be useful to have a counter to know how many certificates do we commit every time

The `committed_certificates` metric is actually tracking this - this has been recently added

> Is this for every certificate or for every leader certificate. I think timings to commit leader certificates is the most interesting

This refers to every certificate. Very good point on the leader commit. We do have a metric , the `commit_rounds_latency` (you can see that on line `33`) which tracks the latency between two successful two leader elections. But I believe I can amend the `certificate_commit_latency` and add a label to tag the "leaders" so we can also easily filter the leader election latency